incubator/virtualcluster/pkg/syncer/vnode/vnode.go

```diff
@@ -66,6 +67,11 @@ func NewVirtualNode(provider provider.VirtualNodeProvider, node *v1.Node) (vnode
 	labels := map[string]string{
 		constants.LabelVirtualNode: "true",
 	}
+
+	if featuregate.DefaultFeatureGate.Enabled(featuregate.SuperClusterPooling) {
```

Comment:
Do you for see this needing anything else specific to pooling? for pooling it must be assumed that all nodes are addressable to the meta tenants, would there be plans other things we need to add later to support additional features.

Reply:
For vnode object, this is the only change I found necessary so far.  Note that it is not necessary that all nodes are addressable to the meta cluster. For example, in edge computing, the nodes and apiserver can be in different networks under the the help of a tunnel service to connect kubelet and apiserver.  Here it is necessary that meta cluster can address the super apiserver, which is already ensured by the syncer since it cannot be started otherwise.

Reply:
Oh, that's awesome.